src/webhint-theme/source/js/scanner-submit.js

```diff
@@ -313,15 +313,24 @@
         document.querySelector('link[rel="icon"]').setAttribute('href', exec[1]);
     };
 
+    var retries = 0;
+
     var queryAndUpdate = function () {
         var callback = function (err, response) {
             if (err) {
-                clearInterval(timer);
-                console.error(err);
+                retries++;
+
+                if (retries >= 5) {
```

Comment:
@molant this is not going to work as you expected.

You need to add `setTimeout(queryAndUpdate, 5000)`.